Tests/FunctionalTests/NetworkResponseHandlerFunctionalTests.swift

```diff
@@ -277,7 +277,7 @@ class NetworkResponseHandlerFunctionalTests: FunctionalTestBase {
             "    }"
         networkResponseHandler.processResponseOnSuccess(jsonResponse: jsonResponse, requestId: "123")
 
-        let dispatchEvents = getDispatchedEventsWith(type: FunctionalTestConst.EventType.EDGE, source: FunctionalTestConst.EventSource.RESPONSE_CONTENT)
+        let dispatchEvents = getDispatchedEventsWith(type: FunctionalTestConst.EventType.EDGE, source: "state:store")
```

Comment:
Can you include a few new tests with 1) handle type empty and 2) no handle type, and verify an event with source `FunctionalTestConst.EventSource.RESPONSE_CONTENT` is dispatched?

Reply:
Added.